tasks/managed/update-trusted-tasks/update-trusted-tasks.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-trusted-tasks
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update trusted-tasks list bundles
+  params:
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the Snapshot spec in the config workspace to apply the mapping to
```

Comment:
> in the config workspace

But there is no config workspace, it's called `data`. Also, to apply what mapping? It seems it's copied from apply-mapping or something.